amazon/ion/simpleion.py

```diff
@@ -50,112 +90,142 @@
-        default: NOT IMPLEMENTED
-        use_decimal: NOT IMPLEMENTED
-        namedtuple_as_object: NOT IMPLEMENTED
-        tuple_as_array: NOT IMPLEMENTED
-        bigint_as_string: NOT IMPLEMENTED
-        sort_keys: NOT IMPLEMENTED
-        item_sort_key: NOT IMPLEMENTED
-        for_json: NOT IMPLEMENTED
-        ignore_nan: NOT IMPLEMENTED
-        int_as_string_bitcount: NOT IMPLEMENTED
-        iterable_as_array: NOT IMPLEMENTED
         tuple_as_sexp (Optional[True|False]): When True, all tuple values will be written as Ion s-expressions.
             When False, all tuple values will be written as Ion lists. Default: False.
-        omit_version_marker (Optional|True|False): If binary is False and omit_version_marker is True, omits the
+        omit_version_marker (Optional[True|False]): If binary is False and omit_version_marker is True, omits the
             Ion Version Marker ($ion_1_0) from the output.  Default: False.
-        **kw: NOT IMPLEMENTED
 
+    Returns None.
+    """
+    if c_ext and __IS_C_EXTENSION_SUPPORTED and (imports is None and indent is None):
+        return dump_extension(obj, fp, binary=binary, sequence_as_stream=sequence_as_stream,
+                              tuple_as_sexp=tuple_as_sexp, omit_version_marker=omit_version_marker)
+    else:
+        return dump_python(obj, fp, imports=imports, binary=binary, sequence_as_stream=sequence_as_stream,
+                           indent=indent,
+                           tuple_as_sexp=tuple_as_sexp, omit_version_marker=omit_version_marker)
+
+
+def dumps(obj, imports=None, binary=True, sequence_as_stream=False,
+          indent=None, tuple_as_sexp=False, omit_version_marker=False):
+    """Serialize obj as described by dump, return the serialized data as bytes or unicode.
+
+    Returns:
+         Union[str|bytes]: The string or binary representation of the data.  if ``binary=True``, this will be a
+             ``bytes`` object, otherwise this will be a ``str`` object
+    """
+    ion_buffer = io.BytesIO()
+
+    dump(obj, ion_buffer, imports=imports, sequence_as_stream=sequence_as_stream, binary=binary,
+         indent=indent, tuple_as_sexp=tuple_as_sexp, omit_version_marker=omit_version_marker)
+
+    ret_val = ion_buffer.getvalue()
+    ion_buffer.close()
+    if not binary:
+        ret_val = ret_val.decode('utf-8')
+    return ret_val
+
+
+def load(fp, catalog=None, single_value=True, parse_eagerly=True, text_buffer_size_limit=None):
+    """Deserialize Ion values from ``fp``, a file-handle to an Ion stream, as Python object(s) using the
+    conversion table described in the pydoc. Common examples are below, please refer to the
+    [Ion Cookbook](https://amazon-ion.github.io/ion-docs/guides/cookbook.html) for detailed information.
+
+    Read an Ion value from file_handle:
+        ``simpleion.load(file_handle)``
+
+    Read Ion values using an iterator:
+        ``
+        it = simpleion.load(file_handle, parse_eagerly=True)
```

Comment:
```suggestion
        it = simpleion.load(file_handle, parse_eagerly=False)
```
Right?

Reply:
Good catch. It was like that before I moved it, but I should've caught that myself. Thanks!